### DISCUSSION TITLE
raspberry pi (32bit) not working (Fixed)

### DISCUSSION THREAD

**Discussion by d0nny94** [haugene/docker-transmission-openvpn#1994] [General]
**hello struggling to get windscibe to work can someone just have a check over and see if they can see an error Thanks**

**Parameters**
     - OPENVPN_PROVIDER=windscribe # Or other compatible OpenVPN provider
      - OPENVPN_CONFIG=Adelaide-Lofty-tcp # Or other region that supports port forwarding. Check with your VPN provider
      - OPENVPN_USERNAME=my openvpn user # VPN provider username
      - OPENVPN_PASSWORD=Myopen vpn pas # VPN provider password

**ERROR**
Running with VPN_CONFIG_SOURCE auto

No bundled config script found for WINDSCRIBE. Defaulting to external config

Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.F4RWEHDKnL

curl: (28) Timeout was reached

./etc/openvpn/fetch-external-configs.sh: line 16: config_repo_temp_dir: unbound variable

Supplied config Adelaide-Lofty-tcp.ovpn could not be found.

Your options for this provider are:

NB: Remember to not specify .ovpn as part of the config name.

Starting container with revision: 6ee67f8210d7f9ca17677c2b108390acc91ec5ab

/etc/openvpn/start.sh: line 32:    10 Aborted                 (core dumped) nslookup ${HEALTH_CHECK_HOST:-"google.com"} > /dev/null 2>&1

WARNING: initial DNS resolution test failed

Creating TUN device /dev/net/tun

Using OpenVPN provider: WINDSCRIBE

Running with VPN_CONFIG_SOURCE auto

No bundled config script found for WINDSCRIBE. Defaulting to external config

Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.JPR7nSUrPw

curl: (28) Resolving timed out after 1225727512802 milliseconds

./etc/openvpn/fetch-external-configs.sh: line 16: config_repo_temp_dir: unbound variable

Supplied config Adelaide-Lofty-tcp.ovpn could not be found.

**Anyhelp would be great Thanks**

**Comment by pkishino** [comment]
Pretty sure it’s windscribe with an R

**Reply by d0nny94** [reply]
appologies did a typo when i made the discussion typo Repaired but error still the same 
i have also tried to fork repo and added my file from windscribe but i seem to get same error
please could you have another look over Thanks donny

**Reply by pkishino** [reply]
Dns is failing.. are you blocking dns or some unusual network setup? What’s your complete docker config?

**Reply by d0nny94** [reply]
Thanks for the help
its running via portainer 
stack is as followed

---
version: "2"
services:
  transmission-openvpn:
    image: haugene/transmission-openvpn
    container_name: transmission-openvpn
    restart: always
    cap_add:
      - NET_ADMIN # This runs the container with raised privileges
    devices:
      - /dev/net/tun # This creates a tunnel for Transmission
    volumes:
      - /srv/dev-disk-by-uuid-ed289bc6-a6e6-4c80-af1d-3e31eb016886/Torrents:/data # Change this to your Torrent path
    environment:
      - OPENVPN_PROVIDER=windscribe # Or other compatible OpenVPN provider
      - OPENVPN_CONFIG=Adelaide-Lofty-tcp # Or other region that supports port forwarding. Check with your VPN provider
      - OPENVPN_USERNAME=removed
      - OPENVPN_PASSWORD=removed
      - LOCAL_NETWORK=192.168.8.0/24 # If your server's IP address is 192.168.1.x, then use this. If your server's IP address is 192.168.0.x, then change to 192.168.0.0/24
      - PUID=998 #Change to your PUID
      - PGID=100 #Change to your PGID
    ports:
      - 9091:9091 #GUI Port
      - 8888:8888 #Transmission Port
    dns:
      - 8.8.8.8 #Use whatever DNS provider you want. This is Google.
      - 8.8.4.4 #Use whatever DNS provider you want. This is Google.

**Reply by pkishino** [reply]
Try removing the Google dns part and see if dns resolved..

**Reply by pkishino** [reply]
Raspberry pie running 32 or 64 but code? There is a known issue with 32 if
you look at the second pinned issue post, but there is a fix afaik.
If you post the whole log output I can tell if that’s the problem..
if you open a console into the container, what does /etc/resolv.conf show?
Can you Ping Google 8.8.8.8 ?

**Reply by d0nny94** [reply]
ahh i see its on 32 bit code issue is the fix runs 3.3 but 3.3 is before windscribes config update so its a checkmate ill have a look at redoing my pi as 64 bit hopefully that works
Thanks for all your help

**Reply by pkishino** [reply]
No, there is a fix running latest and installing some libs on there raspberry.. let me see if I find the link

**Comment by pkishino** [comment]
Look at bottom of this linked issue
https://github.com/alpinelinux/docker-alpine/issues/98

**Reply by d0nny94** [reply]
thanks bud i will look into it see if i can get it working

**Reply by d0nny94** [reply]
yeap works perfectly thanks sir have you got anywhere i can send a small donation for your time 
Thanks
